src/components/home.js

```diff
@@ -32,12 +33,46 @@ function Home(props) {
           setLastUpdated(response.data.statewise[0].lastupdatedtime.slice(0, 15)+response.data.statewise[0].lastupdatedtime.slice(18));
           setDeltas(response.data.key_values[0]);
           setFetched(true);
+          getSummaryData(response.data);
         })
         .catch((err)=>{
           console.log(err);
         });
   };
 
+  const getSummaryData = (data) => {
+    // This should be moved to utils
+    let confirmed = 0;
+    let recoveries = 0;
+    let deaths = 0;
+    data.statewise.map((state, index) => {
+      if (index !== 0) {
+        confirmed += parseInt(state.confirmed);
+        recoveries += parseInt(state.recovered);
+        deaths += parseInt(state.deaths);
+      }
+    });
+
+    const todayDate = data.key_values[0].lastupdatedtime.split(',')[0];
+    const todayData = data.cases_time_series.filter((x) => x.date.trim() === todayDate)[0];
```

Comment:
Same here since you use that data, you will get last updated at's data and not today's data.